Core/src/utils/Handler.cs

```diff
@@ -57,10 +67,37 @@ public class DirHandler
 		 * pc initiaized before each application starts, and if not
 		 * fix any missing dirs
 		 */
-		
 
-		/* create project dir structure */
-		//public static void createProject(String project_name, String project_path );
+
+		/* Create Project Directories */
+
+		public void CreateProject(string projectName, string filePath) {
+
+			string projectDirectory = Path.Combine(filePath, projectName);
+			Directory.CreateDirectory(projectDirectory);
+	
+			Dictionary<string, int> directories = new Dictionary<string, int>() {
```

Comment:
`bool` is preferred instead of `int` for better understanding. but use some other way to represent the directory structure, to reuse and reflector code. (including `subDirectories`)

Reply:
What do you mean by other ways to represent?

Reply:
create a separate class which represent the dir structure and it could be used in many places where you might interact with those dirs.

Reply:
Okay